secrets/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "libcrux-secrets"
+version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+edition.workspace = true
+repository.workspace = true
+readme.workspace = true
+description = "Libcrux utilities for working with secrets"
+exclude = ["/proofs"]
+
+[dependencies]
+hax-lib = { version = "0.2.0", git = "https://github.com/cryspen/hax/" }
```

Comment:
I think v0.2 of hax-lib is published - does this need to be a git dependency?

Reply:
Switched to workspace

Reply:
That one also has git as a source :) But it looks like that hasn't prevented us from publishing, so maybe that's fine.